lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -216,13 +217,23 @@ public SystemAccessControl create(Map<String, String> config)
             requireNonNull(config, "config is null");
 
             Bootstrap bootstrap = new Bootstrap(
-                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class));
+                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class),
+                    conditionalModule(
+                        FileBasedAccessControlConfig.class,
+                        systemAccessControlConfig -> systemAccessControlConfig.getSecurityConfigSource() == LOCAL,
+                        new LocalSystemAccessControlModule()),
+                    conditionalModule(
+                        FileBasedAccessControlConfig.class,
+                        systemAccessControlConfig -> systemAccessControlConfig.getSecurityConfigSource() == REST,
+                        new RestSystemAccessControlModule()));
+
```

Comment:
Can we have them as a part of `FileBasedAccessControlModule`

Reply:
Yeah !! For instead of having the modules in `FileBasedAccessControl` - can we have them in `FileBasedAccessControlModule`